media-api/app/lib/elasticsearch/ElasticSearch.scala

```diff
@@ -125,7 +125,8 @@ object ElasticSearch extends ElasticSearchClient with SearchFilters with ImageFi
       }
   }
 
-  def labelSiblingsSearch(structuredQuery: List[Condition], excludeLabels: List[String] = Nil)(implicit ex: ExecutionContext): Future[AggregateSearchResults] = {
+  def labelSiblingsSearch(structuredQuery: List[Condition], excludeLabels: List[String] = Nil, size: Int = 20)
```

Comment:
I've added it as a param here, but don't want to open up the API to have a infinite number (like the rest of our API :scream:)